echopype/core.py

```diff
@@ -30,7 +30,7 @@
 def init_ep_dir():
     """Initialize hidden directory for echopype"""
     if not ECHOPYPE_DIR.exists():
-        ECHOPYPE_DIR.mkdir()
+        ECHOPYPE_DIR.mkdir(exist_ok=True)
```

Comment:
I have looked into this and online it appears that this is the proper way to handle a race condition that occurs with multiple workers. This looks good!